src/JKang.IpcServiceFramework.Core/IpcRequest.cs

```diff
@@ -1,17 +1,45 @@
-﻿using System;
```

Comment:
Seems like this file has been modified by some automatic tool.
Consider cancel the modification

Reply:
Sorry, force of habit that I hand commented the file to conform to StyleCop rules.  The only mod was to remove the underscore from the private field _genericArguments in IpcRequest.cs.   In c++ underscores were always used but are no longer considered best practice in c#.

If you really want I will withdraw the pull request but please consider keeping these comments and optionally adding more StyleCop comments to your solution in the future.  They really go a long way to help self document the code.

Just FYI, this is a bug fix, not an enhancement :)

This is my first time contributing so please forgive any missteps.  If there is a preferred way, please let me know or point me to some reference documents and I will abide by the rules as I do have several enhancements already implemented that you may be interested in.

Please advise.

Reply:
@hoochharp, Thanks for your contribution.
FYI prefixing private fields with underscore is not really considered as bad practice in C#. 
Microsoft's naming guidelines only apply to public & protected fields (https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members)
Besides coding style refactoring is better be applied globally to avoid inconsistency, and in a seperate PR.

Otherwise this is an open source repository and the copyright claims header should really be removed. :)